README.md

```diff
@@ -26,20 +26,20 @@ import Sentry
 
 // ....
 
-SentrySDK.start(options: [
-    "dsn": "___PUBLIC_DSN___",
-    "debug": true // Helpful to see what's going on
-])
+SentrySDK.start { options in
+    options.dsn = "___PUBLIC_DSN___"
+    options.debug = true // Helpful to see what's going on
+}
 ```
 
 ```objective-c
 @import Sentry;
 
 // ....
 
-[SentrySDK startWithOptions:@{
```

Comment:
Can we leave the other example for Obj-C? The map one is more convenient here we pass only DSN and debug.

Reply:
For Swift, we also pass only DSN and debug. I would disagree that the map one is more convenient, because you don't have code completion. Furthermore, it would be different from Swift. I would keep it as it is.